Makefile

```diff
@@ -1,6 +1,7 @@
 
 # Image URL to use all building/pushing image targets
-IMG ?= kubevirt-ipam-controller:latest
+# Default to registry path for consistency with CI/CD
+IMG ?= ghcr.io/kubevirt/ipam-controller:latest
```

Comment:
maybe we want to be consistent w/ the other branches and consume the branch based flavor ?